clusterman/kubernetes/kubernetes_cluster_connector.py

```diff
@@ -270,14 +270,14 @@ def create_node_migration_resource(
 
     def _evict_or_delete_pods(self, pods: List[KubernetesPod], disable_eviction: bool) -> bool:
         all_done = True
-        logger.info(f"{len(pods)} pods being evicted")
+        logger.info(f"{len(pods)} pods being evicted/deleted")
         for pod in pods:
             try:
                 if disable_eviction:
                     self._delete_pod(pod)
                 else:
                     self._evict_pod(pod)
-                logger.info(f"{pod.metadata.name} ({pod.metadata.namespace}) was deleted")
+                logger.info(f"{pod.metadata.name} ({pod.metadata.namespace}) was evicted/deleted")
```

Comment:
suggestion: would be nice to have this include the node name - but that might be annoying to pull out so no worries :)